packages/e2e-test-utils-playwright/src/editor/update-post.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Publishes the post, resolving once the request is complete (once a notice
```

Comment:
```suggestion
 * Updates the post, resolving once the request is complete (once a notice
```

@Mamaduka or @kevin940726, can you confirm everything looks correct regarding adding a new method to the Editor class?

Reply:
It does look good to me 👍 . The [general guidline](https://developer.wordpress.org/block-editor/contributors/code/testing-overview/e2e/#dont-overload-test-utils-inline-simple-utils) is to avoid adding new utilities when possible, but I think post updating is a common action.

Reply:
Thanks for taking a look! The comment has been updated.

Reply:
FYI: The same updatePost util was discussed in the tests for block bindings: [link](https://github.com/WordPress/gutenberg/pull/58676#discussion_r1478422153). In the end it was decided to keep it as a test util because it would require a deeper discussion.

Reply:
Thanks for the link, @SantosGuillamot! I remembered a similar discussion but couldn't find it.

Reply:
No problem! It makes sense to me as well 🙂 We might need to adapt it to the suggested changes there as the naming.